src/main/java/frc/robot/commands/DriveWithTime.java

```diff
@@ -0,0 +1,79 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DriveBase;
+import edu.wpi.first.wpilibj.Timer;
+
+
+
+public class DriveWithTime extends CommandBase {
+
+  // Initialization of the variables which will allow imported variables to be public(making usable throughout the DriveWithTime command).
+  public double seconds = 0;
+  public double leftSpeeds= 0;
+  public double rightSpeeds = 0;
+
+/**
+ * This command will take in speeds (left and right) and time, for which is will set the motors to the x speed for y seconds.
+ * 
+ * @param seconds        amount of seconds the robot motors will be activated for  
+ * @param leftSpeeds     when the motor is activated, this is the speed at which the LEFT motors will be. speed can be inbetween -1 to 1.
+ * @param rightSpeeds    when the motor is activated, this is the speed at which the RIGHT motors will be. speed can be inbetween -1 to 1.
+ *
+ */
+
```

Comment:
Nitpick: can you remove this space so that the comments feel more associated with the code? In general this file seems to have a few too many extra empty lines.